Tasks/AzurePowerShellV5/azurepowershell.ts

```diff
@@ -59,6 +58,10 @@ async function run() {
         // Generate the script contents.
         console.log(tl.loc('GeneratingScript'));
         let contents: string[] = [];
+
+        const makeModuleAvailableScriptPath = path.join(path.resolve(__dirname), 'TryMakingModuleAvailable.ps1');
+        contents.push(`${makeModuleAvailableScriptPath} -targetVersion '${targetAzurePs}' -platform Linux`);
```

Comment:
this will be executed even when user selects latest. that time targetAzurePs will have "" value.
unzipping/ downloading modules logic is not required for latest

Reply:
Yes this is intentional as we handle the same in the respective script as well as we do logging in that scenario to mark that for moduleSource as folder.